aspnetcore/fundamentals/dependency-injection.md

```diff
@@ -560,3 +560,4 @@ Dependency injection is an *alternative* to static/global object access patterns
 * [Explicit Dependencies Principle](https://deviq.com/explicit-dependencies-principle/)
 * [Inversion of Control Containers and the Dependency Injection Pattern (Martin Fowler)](https://www.martinfowler.com/articles/injection.html)
```

Comment:
Dependency Injection --> DI
- this suggestion should be applied all throughout the doc (perhaps in a separate PR)

Reply:
The title of the linked content is ...

> Inversion of Control Containers and the Dependency Injection pattern

Reply:
> perhaps in a separate PR

Yes, this wasn't a UE pass. The goal of the PR is just to resolve the issue.